docs/chapters/baseplatform/creating-sdk.rst

```diff
@@ -0,0 +1,14 @@
+Building the SDK installer
+===========================
+
+Building the SDK installer is not very different from creating the corresponding
+target image. All you have to do, is pass an additional option to bitbake: 
```

Comment:
It's grammatically correct AFAIK and when it's correct, it's always best to use a comma than not. :)